app/src/main/java/com/github/scribeWizTeam/scribewiz/Fragments/ProfilePageFragment.kt

```diff
@@ -55,35 +66,34 @@ class ProfilePageFragment(contentLayoutId: Int) : Fragment(contentLayoutId) {
     constructor() : this(0) {
         // Default constructor
     }
-
     @Composable
     fun ProfilePage(){
         val context = LocalContext.current
 
-        // Get the current user
-        val user = FirebaseAuth.getInstance().currentUser
-
-        // Retrieve the database
-        val db = Firebase.firestore
-
         // Check if user is logged in, set default values otherwise
         var userName = "Guest"
         var numRecordings = "0"
```

Comment:
I can see that this is a demonstration of the utility provided by friendsList, this hardcoded friends list should not be present in the final revision.